client/my-sites/stats/stats-comments/index.jsx

```diff
@@ -52,72 +56,76 @@ export default React.createClass( {
 	},
 
 	changeFilter( selection ) {
-		let gaEvent;
 		const filter = selection.value;
+		if ( filter === this.state.activeFilter ) {
+			return;
+		}
+		let gaEvent;
+		switch ( filter ) {
+			case 'top-authors':
+				gaEvent = 'Clicked By Authors Comments Toggle';
+				break;
+			case 'top-content':
+				gaEvent = 'Clicked By Posts & Pages Comments Toggle';
+				break;
+		}
 
-		if ( filter !== this.state.activeFilter ) {
-			switch ( filter ) {
-				case 'top-authors':
-					gaEvent = 'Clicked By Authors Comments Toggle';
-					break;
-				case 'top-content':
-					gaEvent = 'Clicked By Posts & Pages Comments Toggle';
-					break;
-			}
-
-			if ( gaEvent ) {
-				analytics.ga.recordEvent( 'Stats', gaEvent );
-			}
-
-			this.setState( {
-				activeFilter: filter
-			} );
+		if ( gaEvent ) {
+			this.props.recordGoogleEvent( 'Stats', gaEvent );
 		}
+
+		this.setState( {
+			activeFilter: filter
+		} );
 	},
 
 	updateHeaderState( options ) {
 		this.setState( options );
 	},
 
 	renderCommentFollowers() {
-		const { commentFollowersList, site } = this.props;
+		const { commentFollowersData, siteSlug, translate, numberFormat } = this.props;
```

Comment:
What do you think about passing in `commentFollowersData.total` as a stand-alone prop - maybe call it `totalCommentFollowers` to simplify the logic in this method?